ui/elements/Carousel/Carousel.styles.ts

```diff
@@ -49,9 +49,13 @@ export const StyledCarouselItemsContainer = styled.div<StyledCarouselItemsContai
     width: 100%;
     display: flex;
     gap: ${(props) => props.$gap}px;
+    overflow-x: auto;
     scroll-behavior: smooth;
-    transform: translateX(${(props) => props.$translateX}px);
-    transition: transform 0.5s ease-in-out;
+    -webkit-overflow-scrolling: touch;
+    scrollbar-width: none;
+    &::-webkit-scrollbar {
+        display: none;
+    }
```

Comment:
let us not hide the scrollbar here, we can leave this to the authors of projects which are using this library